File: gossip-board-ui-template/src/app/models/poll-post.ts
import {User} from './user';
import {PollItem} from './poll-item';

export class PollPost {
  id: number;
  postAuthor: User;
  subject: string;
  description: string;
  creationDate: Date;
  pollItems: PollItem[];
  likes: any;
}

File: gossip-board-template/GossipBoard/Models/Post/Poll/PollPost.cs
using System.Collections.Generic;

namespace GossipBoard.Models.Post.Poll
{
    public class PollPost : Post
    {
        public List<PollItem> PollItems { get; set; }
        public List<PollPostLike> Likes { get; set; }

        public PollPost()
        {
            PollItems = new List<PollItem>();
            Likes = new List<PollPostLike>();
        }
    }
}